app/controllers/organizations/staff/external_form_upload_controller.rb

```diff
@@ -4,18 +4,20 @@ class ExternalFormUploadController < Organizations::BaseController
       layout "dashboard"
 
       def index
-        authorize! :external_form_upload,
-          context: {organization: Current.organization}
+        authorize! :external_form_upload, context: {organization: Current.organization}
       end
 
       def create
-        authorize! :external_form_upload,
-          context: {organization: Current.organization}
-        file = params[:files]
+        authorize! :external_form_upload, context: {organization: Current.organization}
 
         # Only processes single file upload
```

Comment:
We limit the import field to only accept one file, too. I am sure you noticed this.

Reply:
100%. I added the comment because it may be confusing to see `params[:files]` (plural). The front end currently handling the allowing only one file. Is there a need for some backend control over that? If the front end was bypassed it should raise an error in the service.

Reply:
hmm we could add a before_action in the controller to check the length of the files array and do X if it is >1

Reply:
Given it could be used in multiple controllers, makes sense to add in a controller concern that can be included as needed.